### PR TITLE
Hackinrish/button

### LINKED ISSUE #8: Create Button
image

### PR DESCRIPTION
**Pull request by hackinrish**
Screenshot 2024-10-21 at 6 43 38 PM

- Button that takes in a text prop and an onclick functon, function can be modified later for according use
- Closes Issue #8

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,8 +1,6 @@
 const Home = () => {
   return (
-    <div className="flex h-screen w-screen items-center justify-center">
```
Comment (jnalbert) [inline src/app/page.tsx]: Can you revert this change so it still says hello world

File: src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Link from "next/link";
+import React from "react";
+
+interface ButtonProps {
+  text: string;
+  link: string;
```
Comment (jnalbert) [inline src/components/Button.tsx]: instead of passing in a link can you pass in an arrow function here. For example it can be `handleSubmit: () => void`. Then this function should be called when the button component is clicked.

File: src/components/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+const Button = ({ text, link }: ButtonProps) => {
+  return (
+    <div className="rounded-md bg-blue-900 px-8 py-2 font-serif text-[vw] text-white">
```
Comment (jnalbert) [inline src/components/Button.tsx]: Instead of a div can we use the button component here. It's better for accessibility and makes more sense in this case.

Comment (jnalbert) [inline src/components/Button.tsx:11]: Instead of a div can we use the button component here. It's better for accessibility and makes more sense in this case.

File: src/components/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+interface ButtonProps {
+  text: string;
+  onClick: (params: string) => void;
+}
+
+const Button: React.FC<ButtonProps> = ({ text, onClick }) => {
+  return (
+    <button
+      className="rounded-md bg-blue-900 px-8 py-2 font-serif text-[vw] text-white"
+      onClick={() => onClick(text)}
```
Comment (jnalbert) [inline src/components/Button.tsx]:
```suggestion
      onClick={onClick}
```

File: src/components/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+interface ButtonProps {
+  text: string;
+  onClick: (params: string) => void;
```
Comment (jnalbert) [inline src/components/Button.tsx]: remove the params from this function

**Review by jnalbert — COMMENTED** [review]
Overall looks good just some small edits

**Review by shahdivyank — APPROVED** [review]
